core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -152,8 +152,10 @@ public Deadline getDeadline() {
 
   /**
    * Returns a new {@code CallOptions} with attributes for affinity-based routing.
+   *
+   * @deprecated use {@link #withOption(Key, Object)}.
    */
-  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1766")
+  @Deprecated
```

Comment:
Go ahead and leave the `@ExperimentalApi` annotation, to show that we can actually delete it.